.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "norvid"]
+	path = norvid
+	url = git@git.vauxoo.com:vauxoo/norvid.git
```

```suggestion
	url = git@git.vauxoo.com:vauxoo/norvid.git
	branch = 18.0
```

this will assigned automatically if you execute `git submodule add -b 18.0 git@git.vauxoo.com:vauxoo/norvid.git`

---

Hi @rolandojduartem change applied